pkg/tagger/collectors/docker_extract_test.go

```diff
@@ -348,6 +348,26 @@ func TestDockerExtractImage(t *testing.T) {
 				"image_tag:latest",
 			},
 		},
+		{
+			testName: "Some Nomad Setup",
+			co: types.ContainerJSON{
+				ContainerJSONBase: &types.ContainerJSONBase{
+					Image: "sha256:380b233f1574da39494e2b36e65f262214fe158af5ae7a94d026b7a4e46fa358",
+				},
+				Config: &container.Config{
+					Image: "quay.io/hashicorp/cloud-consul-ama:3451-be4c56f",
```

Comment:
Hi there! Is it possible to use more anonymized repos/orgs here and below? Thanks!

Reply:
Of course, done!